src/heartRate.cpp

```diff
@@ -57,6 +57,23 @@
 
 #include "heartRate.h"
 
+#include <Filters.h>
+#include <Filters/Butterworth.hpp>
+
+#ifdef EMOTIBIT_PPG_100HZ
+// Sampling frequency
+const double f_s = 100; // Hz
+#else
+// Sampling frequency
+const double f_s = 25; // Hz
+#endif
+// Cut-off frequency (-3 dB)
+const double f_c = 3; // Hz
```

Comment:
For Reviewer: 
This frequency was chosen based on "how the filtered output looked".

We should improve on this as a part of a future update.

Reply:
Added a issue in the FW repo. 
https://github.com/EmotiBit/EmotiBit_FeatherWing/issues/258